ovs-p4rt/sidecar/testing/ipv4_tunnel_test.h

```diff
@@ -35,9 +35,9 @@ class Ipv4TunnelTest : public TableEntryTest {
         << "Error converting " << IPV4_DST_ADDR;
     info.remote_ip.prefix_len = IPV4_PREFIX_LEN;
 
-    info.src_port = htons(SRC_PORT);
-    info.dst_port = htons(DST_PORT);
-    info.vni = htons(VNI);
+    info.src_port = SRC_PORT;
```

Comment:
The `info` struct is what's normally passed in by OVS. Fields are assumed to be in native byte order.